x-pack/plugins/fleet/server/services/settings.ts

```diff
@@ -84,6 +84,6 @@ export function getCloudFleetServersHosts() {
     }
 
     // Fleet Server url are formed like this `https://<deploymentId>.fleet.<host>
-    return [`https://${cloudSetup.deploymentId}.fleet.${res.host}`];
+    return [`https://${cloudSetup.deploymentId}.fleet.${res.host}:${res.defaultPort}`];
```

Comment:
I guess there isn't a case where `res.defaultPort` is undefined?

Reply:
No if it's not defined in the cloud ID it will `443`